hwdb/60-keyboard.hwdb

```diff
@@ -618,6 +618,21 @@ evdev:atkbd:dmi:bvn*:bvr*:svnHP*:pnHP*ProBook*645*G4*
  KEYBOARD_KEY_73=slash                                  # Slash key
  KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
 
+##########################################################
+# Huawei
+##########################################################
+
+# Huawei WMI hotkeys driver
+evdev:name:Huawei WMI hotkeys:dmi:bvn*:bvr*:bd*:svnHUAWEI*
+ KEYBOARD_KEY_287=FW250                                 # Microphone mute button, should be micmute
```

Comment:
Maybe, `micmute` instead of `FW250`?

Reply:
It was supposed to be 'f20'. Sorry for the typo.